index.js

```diff
@@ -102,6 +102,7 @@ const plugins = [
       { no: "[Rr][Ff][Cc]\\d+", yes: "RFC <number>" },
       { yes: "Unix" },
       { yes: "V8" },
+      { no: "valgrind", yes: "Valgrind" },
```

Comment:
Thanks for the PR. Two comments:

1. We don't need the `no` entry in this case. Just the `yes` is enough (and will prevent people from doing other things like `ValGrind` or `VALGRIND`). 
2. The entries are in alphabetical order. Can you move this one up one line (so it's after `Unix` and before `V8`)?

Reply:
Can do, and will do! :)